src/wdio/services/TerraService.js

```diff
@@ -64,7 +93,9 @@ export default class TerraService {
   before() {
     global.expect = chai.expect;
     global.Terra = {
-      viewports,
+      viewports: getViewports,
+      resize,
+      should,
```

Comment:
This is a total nit, but it might be easier to reference the should helpers if defined like:

global.Terra = {
    should: {
          beAccessible,
          beComparable,
    }
}

I'm thinking in terms of consumers checking which helpers are available and if we were to add more helpers in the future.